### PR TITLE
feat: fitItems function

### PR DESCRIPTION
**Pull request by Blakko**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] commit message follows commit guidelines

##### Description of change

This PR implements the `fitItems` function. This allows to fit a subset of the graph, eg: fit only the selected items.
This is useful with big graphs, where you want to have a closer look at a bunch of nodes.

In this example I'm using the `fitItems` function on the selected nodes via a keyboard shortcut:

`animate` false

https://user-images.githubusercontent.com/2861371/182404337-01167ecc-acfc-4ec2-98ce-c9af4da3b1e7.mp4

`animate` true

https://user-images.githubusercontent.com/2861371/182404386-77fbaccb-3fdb-4167-9bc6-ff2735257def.mp4

### CODE REVIEW

File: packages/core/src/graph/controller/view.ts [hunk 1]
```diff
@@ -184,6 +184,73 @@ export default class ViewController {
     }
   }
 
+  public fitItems(items: Item[], animate?: boolean, animateCfg?: GraphAnimateConfig): void {
```
Comment (Yanyan-Wang) [inline packages/core/src/graph/controller/view.ts:187]: There is a function named `focusItem`, which translates the graph to make the focused item align to the view center. Is it possible to make `fitItems` to be `focusItems`? Maybe add a parameter to control whether zoom the graph or just translate the graph to focus on the items. e.g.

public focusItems(items: Item[], zoomToFit?: boolean, animate?: boolean, animateCfg?: GraphAnimateConfig)

File: packages/core/src/graph/graph.ts [hunk 2]
```diff
@@ -641,6 +641,22 @@ export default abstract class AbstractGraph extends EventEmitter implements IAbs
...
+  /**
+   * Fits the passed items into the view
+   * @param {Item[]} items Items you want to fit into the view
+   * @param {Padding} padding padding around the items
+   * @param {boolean} animate Wheter to animate the transition
+   * @param {GraphAnimateConfig} animateCfg Animation configuration
+   */
+  public fitItems(items: Item[], padding?: Padding, animate?: boolean, animateCfg?: GraphAnimateConfig): void {
+    if (padding) {
+      this.set('fitViewPadding', padding);
```
Comment (Yanyan-Wang) [inline packages/core/src/graph/graph.ts:653]: since it is not 'fitView', I think it should not affect the graph's global setting 'fitViewPadding'

**Comment by Yanyan-Wang** [comment]
Hi @Blakko , thanks for your PR! I have commented with some suggestions, please check them out to see if they are reasonable.

**Comment by Blakko** [comment]
My bad, I totally missed the `focusItem` function :)
I'll apply the changes you suggested and update this PR asap!

**Pull request merged**